File: complete_string/complete_string.py
#!/usr/bin/env python

YES = "YES"
NO = "NO"
results = []
num_strings = int(raw_input())

for i in range(num_strings):
  uniq_chars = {x for x in raw_input() if x in 'abcdefghijklmnopqrstuvwxyz'}

  if len(uniq_chars) == 26:
    results.append(YES)
  else:
    results.append(NO)

for r in results:
  print r

File: complete_string/complete_string.pl
#!/usr/bin/env perl -w

use strict;
use warnings;

use constant YES => "YES";
use constant NO => "NO";

my @results;

my @input = <STDIN>;
chomp @input;


my $num_strings = $input[0];

foreach (1..$num_strings) {

  my %h = map { ($_, undef) if (m/[a-z]/) } split //, $input[$_];

  if (scalar keys %h == 26) { 
    push @results, YES;
  } else {
    push @results, NO;
  }

}

print STDOUT join "\n", @results;
print STDOUT "\n";